fudge-ratio/src/main/webapp/code/chooseActivities.js

```diff
@@ -0,0 +1,63 @@
+const dropdownText = `
+<div id="dropdown">
```

Comment:
Each file / component / screen is really simple now, first show the HTML that you are going to put in the screen